### PR TITLE
9 assignment page

### PR DESCRIPTION
**Pull request by kongebra**
# Added
* View-plugins
* Subrouter for `/admin`
* Added middleware for authentication admins
* Display single assignment
* Refactor `form.js` prettier prefixes
* Started on `/admin/settings`
  * Displays "Under development" at the moment
* Split `admin.go` into several files, for better overview

### CODE REVIEW

File: webservice/template/course.tmpl [hunk 1]
```diff
@@ -38,36 +38,19 @@
...
-                                    <div class="card-header">Assignment 3</div>
-                                    <div class="card-body">
-                                        <h5 class="card-title">Light card title</h5>
-                                        <p class="card-text">Some quick example text to build on the card title and make up the bulk of the card's content.</p>
-                                        <a href="user" class="btn btn-outline-dark">Go somewhere</a>
+
+                            {{range .Assignments}}
+                                <div class="col-md-6">
+                                    <div class="card my-1 border-dark text-dark bg-light">
+                                        <div class="card-header">Assignment?</div>
```
Comment (BredeFK) [inline webservice/template/course.tmpl]: Why the '?'

File: webservice/template/course.tmpl [hunk 2]
```diff
@@ -38,36 +38,19 @@
...
+                                        <div class="card-body">
+                                            <h5 class="card-title">{{.Name}}</h5>
+                                            <p class="card-text">{{.Description}}</p>
+                                            <a href="/assignment/{{.ID}}" class="btn btn-outline-dark">See details</a>
```
Comment (BredeFK) [inline webservice/template/course.tmpl]: I get Error 500 from this

File: webservice/template/admin/assignment/single.tmpl [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+            </tr>
+
+            <tr>
+                <td><strong>Course</strong></td>
+                <td>{{.Assignment.CourseID}}</td>
+            </tr>
+
+            <tr>
+                <td><strong>Submission Form</strong></td>
+                <td>{{.Assignment.SubmissionID}}</td>
```
Comment (BredeFK) [inline webservice/template/admin/assignment/single.tmpl]: Maybe make this prettier? but is not a must

File: webservice/template/admin/assignment/single.tmpl [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+            </tr>
+
+            <tr>
+                <td><strong>Review From</strong></td>
+                <td>{{.Assignment.ReviewID}}</td>
```
Comment (BredeFK) [inline webservice/template/admin/assignment/single.tmpl]: Maybe make this prettier? but is not a must

File: webservice/template/course.tmpl (hunk 1 above)
Comment (kongebra) [reply webservice/template/course.tmpl]: Did not know what to put here.

File: webservice/template/course.tmpl (hunk 2 above)
Comment (kongebra) [reply webservice/template/course.tmpl]: Have you installed the new database?

File: webservice/template/admin/assignment/single.tmpl (hunk 3 above)
Comment (kongebra) [reply webservice/template/admin/assignment/single.tmpl]: Non-essential at this moment

File: webservice/template/admin/assignment/single.tmpl (hunk 4 above)
Comment (kongebra) [reply webservice/template/admin/assignment/single.tmpl]: Non-essential at this moment

File: webservice/template/course.tmpl (hunk 2 above)
Comment (BredeFK) [reply webservice/template/course.tmpl]: apparently not :(

**Pull request merged**